docs/ossm/ossm2-migration/README.md

```diff
@@ -8,22 +8,117 @@ Before you begin to migrate your controlplane from OpenShift Service Mesh 2.6 to
+    - Tracing: `spec.tracing.type=None`. Follow [these instructions](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-configuring-distr-tracing-tempo_observability) for instructions on how to configure your `ServiceMeshControlPlane` with OpenShift Distributed Tracing as a replacement for the tracing addon. If not using OpenShift Distributed Tracing, you can add the custom tracing endpoint to your `spec.meshConfig.extensionProviders` configuration and create the corresponding `Telemetry` resource.
+    - Kiali: `spec.addons.kiali.enabled=false`. You will need to create a standalone `Kiali` resource to replace the addon. If you haven't already done this as part of the previous steps for Prometheus or Tracing, follow [these instructions](https://docs.redhat.com/en/documentation/red_hat_openshift_service_mesh/3.0.0tp1/html/observability/kiali-operator-provided-by-red-hat#ossm-install-kiali-operator_ossm-kiali-assembly) on how to install and configure Kiali as a standalone resource.
+    - Grafana: `spec.addons.grafana.enabled=false`. Grafana is not supported in OpenShift Service Mesh 3.0.
+  - Istio OpenShift Routing (IOR) is disabled. Follow [these instructions](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-route-migration) to migrate off of IOR.
+  - Default ingress/egress gateways are disabled. Follow [these instructions](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-gateway-migration) to migrate off of the default ingress/egress gateways.
 - Run the [migration-checker script](migration-checker.sh) to detect any issues with your environment.
 
 > [!WARNING]
 > You must upgrade your OpenShift Service Mesh 2 Operator to the latest release **before** you install the OpenShift Service Mesh 3 operator. If you upgrade your OpenShift Service Mesh 2 operator _after_ you install your OpenShift Service Mesh 3 operator, you will need to then uninstall and reinstall your OpenShift Service Mesh 3 operator to ensure the included CRDs are up to date.
 
+By the end of this checklist, your `ServiceMeshControlPlane` should look something like this:
+
+```yaml
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  version: v2.6 # 1.
+  security: # 2.
+    manageNetworkPolicy: false
+  addons: # 3.
+    grafana:
+      enabled: false
+    kiali:
+      enabled: false
+    prometheus:
+      enabled: false
+  meshConfig: # 4.
+    extensionProviders:
+      - name: prometheus
```

Comment:
Do  you know if this is a default value? It should be right? We are not using this in our automation and it's working fine. Just curious, no need to change anything.

Reply:
Yes but iirc you actually do need to specify `prometheus: {}` in order for the external prometheus to work with 2.6.

Reply:
Not sure, but definitely no harm to have it here.